fastplotlib/plot.py

```diff
@@ -103,4 +107,54 @@ def show(self, autoscale: bool = True):
         if autoscale:
             self.auto_scale(maintain_aspect=True, zoom=0.95)
 
-        return self.canvas
+        if self.toolbar:
+            tools = ToolBar(self).toolbar
+            return VBox([self.canvas, tools])
+        else:
+            return self.canvas
+
+
+class ToolBar:
```

Comment:
Make a separate module for this class, something like "_toolbar.py". You probably want a separate class for the Plot and Gridplot ones.

Reply:
I had it like that initially but kept running into circular import issues because `ToolBar` will encapsulate a `Plot` instance but then `Plot.show()` will need `ToolBar` instance

Reply:
Ah right nevermind